model/src/main/proto/interaction_object.proto

```diff
@@ -53,5 +54,30 @@ message Fraction {
 
 // Structure for a ListOfString object.
 message ListOfSetsOfHtmlStrings {
-  repeated StringList set_of_html_strings = 1;
+  StringList set_of_html_strings = 1;
+}
+
+// Structure for a ImageWithRegions object.
+message ImageWithRegions {
+  string image_path = 1;
+  repeated LabelRegions label_regions = 2;
+}
+
+message LabelRegions {
```

Comment:
Ditto here. Also, I suggest nesting this within the ``ImageWIthRegion`` message since it's always going to correspond to that one. Ditto for ``Region`` and ``NormalizedRectangle2d`` below. Both can be nested within the new interaction object.